Framework/Core/test/test_DataAllocator.cxx

```diff
@@ -93,20 +98,20 @@ DataProcessorSpec getSinkSpec()
       LOG(INFO) << dh->dataOrigin.str << " " << dh->dataDescription.str << " " << dh->payloadSize;
     }
     auto object1 = pc.inputs().get<o2::test::TriviallyCopyable>("input1");
-    assert(*object1 == o2::test::TriviallyCopyable(42, 23, 0xdead));
+    ASSERT_ERROR(*object1 == o2::test::TriviallyCopyable(42, 23, 0xdead));
```

Comment:
The test is not using boost unit test, it's a compiled workflow. I have not looked into the possibility to test a complete workflow as boost unit test. I guess right now there will be a conflict because of the two main functions in the workflow driver and the boost test suite respectively.